From 04a5683cf0c70da085643bc5f13405ad21292289 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kirill=20M=C3=BCller?= <kirill@cynkra.com>
Date: Sat, 30 Nov 2024 22:08:47 +0100
Subject: [PATCH] Avoid memory leak

---
 src/import-file.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/import-file.c b/src/import-file.c
index 34d2b1b8..c2bb19f6 100644
--- a/src/import-file.c
+++ b/src/import-file.c
@@ -92,7 +92,11 @@ RS_sqlite_import(
 
   while ((zLine = RS_sqlite_getline(in, eol)) != NULL) {
     lineno++;
-    if (lineno <= skip) continue;
+    if (lineno <= skip) {
+      free(zLine);
+      zLine = NULL;
+      continue;
+    }
     i = 0;
     azCol[0] = zLine;
     for (i = 0, z = zLine; *z && *z != '\n' && *z != '\r'; z++) {
-- 
2.43.0

