commit 1251fa9dfa37f6386a374235662349ffda4e1893
Author: Kirill Müller <krlmlr+r@mailbox.org>
Date:   Mon Sep 28 15:50:33 2020 +0200

    Tweak

diff --git a/src/vendor/sqlite3/regexp.c b/src/vendor/sqlite3/regexp.c
index 03bbeb9..a1517c8 100644
--- a/src/vendor/sqlite3/regexp.c
+++ b/src/vendor/sqlite3/regexp.c
@@ -747,7 +747,8 @@ static void re_sql_func(
 #ifdef _WIN32
 __declspec(dllexport)
 #endif
-int sqlite3_regexp_init(
+// Keep attribute_visible when upgrading
+attribute_visible int sqlite3_regexp_init(
   sqlite3 *db, 
   char **pzErrMsg, 
   const sqlite3_api_routines *pApi
